modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -61,7 +61,7 @@ The following information provides examples for configuring networking and bondi
 Ordering is important when adding the kernel arguments: `ip=`, `nameserver=`, and then `bond=`.
 ====
 
-The networking options are passed to the `dracut` tool during system boot. For more information about the networking options supported by `dracut`, see the `dracut.cmdline` manual page.
+The networking options are passed to the `dracut` tool during system boot. For more information about the networking options supported by `dracut`, see the link:https://www.man7.org/linux/man-pages/man7/dracut.cmdline.7.html[`dracut.cmdline` manual page].
```

Comment:
So it's not official guidance yet, but we've been working on an update to the SSG regarding referencing man pages, and one of the things it says is to not link to them.

Again, nothing official yet so I don't think you need to change it, just wanted to give a heads up!